server/src/test/java/com/objectcomputing/checkins/services/guild/GuildControllerTest.java

```diff
@@ -76,15 +92,21 @@ void testEmailSentToGuildLeadWhenGuildMembersAdded() {
         List<GuildUpdateDTO.GuildMemberUpdateDTO> newAndExistingMembers = new ArrayList<>();
         newAndExistingMembers.add(newMemberDTO);
         newAndExistingMembers.add(updateDefaultGuildMemberDto(guildLead, guildLead.getLead()));
+        newAndExistingMembers.add(updateDefaultGuildMemberDto(guildLead2, guildLead2.getLead()));
         requestBody.setGuildMembers(newAndExistingMembers);
 
         final HttpRequest<GuildUpdateDTO> request = HttpRequest.PUT("/", requestBody).basicAuth(memberProfileOfAdmin.getWorkEmail(), ADMIN_ROLE);
         client.toBlocking().exchange(request, GuildResponseDTO.class);
 
-        verify(emailSender).sendEmail(null, null,
-                "Membership Changes have been made to the Ninja guild",
-                "<h3>Changes have been made to the Ninja guild.</h3><h4>The following members have been added:</h4><ul><li>Bill Charles</li></ul><a href=\"https://checkins.objectcomputing.com/guilds\">Click here</a> to view the changes in the Check-Ins app.",
-                "billm@objectcomputing.com"
+        assertEquals(2, emailSender.events.size());
+        assertEquals(
+                List.of(
+                        // Email to the Guild lead
+                        List.of("SEND_EMAIL", "null", "null", "Membership changes have been made to the Ninja guild", "<h3>Bill Charles has joined the Ninja guild.</h3><a href=\"https://checkins.objectcomputing.com/guilds\">Click here</a> to view the changes in the Check-Ins app.", memberProfile.getWorkEmail() + "," + memberProfile2.getWorkEmail()),
+                        // Email to both the Guild leads
+                        List.of("SEND_EMAIL", "null", "null", "Membership Changes have been made to the Ninja guild", "<h3>Changes have been made to the Ninja guild.</h3><h4>The following members have been added:</h4><ul><li>Bill Charles</li></ul><a href=\"https://checkins.objectcomputing.com/guilds\">Click here</a> to view the changes in the Check-Ins app.", memberProfile.getWorkEmail() + "," + memberProfile2.getWorkEmail())
+                ),
+                emailSender.events
         );
```

Comment:
I believe this is a bug and we may be sending too many emails

Reply:
Raised https://github.com/objectcomputing/check-ins/issues/2549

Reply:
potential PR to fix is in https://github.com/objectcomputing/check-ins/pull/2551

If that gets merged, we need to remove one of the Lists in this assert (and reduce the size assertion to 1)